app/wama.web/Views/AdminConsole/UserTool/CreateNewUserAccount.cshtml

```diff
@@ -60,6 +60,16 @@
             <label asp-for="Gender"></label>
             <select asp-for="Gender" asp-items="@Html.GetEnumSelectList<Gender>()" class="form-control"></select>
         </div>
+        @{if ((ViewBag.AccountType) == UserAccountType.Patron)
```

Comment:
A better version of this, can be written like this

`<input asp-for="HasBeenApproved" type="hidden" value="@(ViewBag.AccountType == UserAccountType.Patron)" />`

This way, no need for an **_if-else_**, it's also much easier to read.